test/parallel_api/algorithm/alg.sorting/sort.pass.cpp

```diff
@@ -261,18 +422,20 @@ test_sort(Compare compare, Convert convert)
     for (size_t n = 0; n < 100000; n = n <= 16 ? n + 1 : size_t(3.1415 * n))
     {
         LastIndex = n + 2;
+
         // The rand()%(2*n+1) encourages generation of some duplicates.
         // Sequence is padded with an extra element at front and back, to detect overwrite bugs.
-        Sequence<T> in(n + 2, [=](size_t k) { return convert(k, rand() % (2 * n + 1)); });
-        Sequence<T> expected(in);
-        Sequence<T> tmp(in);
+        TestUtils::Sequence<T> in(n + 2, [=](size_t k) { return convert(k, rand() % (2 * n + 1)); });
```

Comment:
I think we don't need TestUtils:: (here and everywhere in the file)  because you added `using namespace TestUtils;` in the begin of the file.

Reply:
Fixed